docker/README.md

```diff
@@ -65,6 +65,10 @@ docker-compose up
 After several minutes for superset initialization to finish, you can open a browser and view [`http://localhost:8088`](http://localhost:8088) 
 to start your journey.
 
+To check the progression of the init procedures run:
+
+`docker logs preset_fork_superset_superset-init_1 -f`
```

Comment:
Is the container actually called "preset_fork_xxx"?

Reply:
work on my laptop, lol

Reply:
named the containers like on #9547